cypress/tests/integration/Jobs.spec.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * @file cypress/tests/integration/Jobs.spec.js
+ *
+ * Copyright (c) 2014-2021 Simon Fraser University
+ * Copyright (c) 2000-2021 John Willinsky
+ * Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
+ *
+ */
+
+ describe('Jobs tests', function() {
+    it('Check if Jobs page is alive and with contents', function() {
+        cy.login('dbarnes', null, 'publicknowledge');
+
+        // Add two test jobs on queue
+        cy.exec('php lib/pkp/tools/jobs.php test');
```

Comment:
What happens if there is a queue runner that handles these? I presume they'll disappear as soon as the queue runner hits them?

Reply:
They're being added into a queue called `queuedTestJob`. I think the probability of this happen is extreme low.

Reply:
Also, the test job always throw an exception, so, he's going back to the queue every time.

Reply:
Would it be possible to assert something based on the specific jobs that were added, rather than assuming that the queue was empty beforehand (and thus the 2 entries must be the test entries)? Then make the `purge` command only clear the `queuedTestJob` queue? That would be a little more error-proof, I think.

Reply:
I've updated the test to handle this checking if the there's two rows with queue as 'QueuedTestJob' on the page. Also, cleaning only the `queuedTestJob` queue after tests.

Reply:
Thanks!